deep_quoridor/src/agents/core/trainable_agent.py

```diff
@@ -182,6 +195,7 @@ def handle_step_outcome_all(self, observation_before_action, action, game, playe
             if state_after_action is not None
             else np.zeros_like(state_before_action.cpu().numpy()),
             float(done),
+            next_state_mask.cpu().numpy(),
```

Comment:
shouldn't all the next_state_mask logic be done only if the mask_targetq flag is true?